tests/pipelines/stable_diffusion_xl/test_stable_diffusion_xl.py

```diff
@@ -146,7 +147,7 @@ def test_stable_diffusion_xl_euler(self):
         assert image.shape == (1, 64, 64, 3)
         expected_slice = np.array([0.5873, 0.6128, 0.4797, 0.5122, 0.5674, 0.4639, 0.5227, 0.5149, 0.4747])
```

Comment:
We'll have to update the values here I'm afraid :-) Good idea to change the number of channels

Reply:
How did the team come with the numbers in the first place?

Reply:
haha, ok. I thought, that maybe calculated the values *manually*, but if just using what I get is an option, then I'll do that :-)